lib/ios/BackpackReactNative/Classes/CalendarBridge/RCTBPKCalendar.m

```diff
@@ -147,5 +147,22 @@ - (void)setRct_colorBuckets:(NSArray<RCTBPKColorBucket *> *_Nullable)rct_colorBu
     [self reloadData];
 }
 
+- (void)setRct_disabledDates:(RCTBPKDateMatcher *_Nullable)rct_disabledDates {
+    if (_rct_disabledDates != rct_disabledDates) {
+        _rct_disabledDates = rct_disabledDates;
+
+        NSMutableArray *localDates = [[NSMutableArray alloc] initWithCapacity:rct_disabledDates.dates.count];
+        for (int j = 0; j < rct_disabledDates.dates.count; j += 1) {
+            localDates[j] = [RCTBPKCalendarDateUtils convertDateToLocal:rct_disabledDates.dates[j]
+                                                          localCalendar:self.gregorian
+                                                            utcCalendar:self.utcCalendar];
+        }
+
+        rct_disabledDates.dates = localDates;
+
+        [self reloadData];
```

Comment:
This call doesn't seem to be causing a visual update as I'd expect. I have no idea why, as it calls `reloadData` on the FSCalendar view